### ISSUE TITLE
All code and pre-trained models

### ISSUE THREAD

**Issue by attutude** [JuanFMontesinos/VoViT#5]
Thank you for your excellent work, can you release all the code and pre-training model, I want to compare in the same batch size.

**Comment by JuanFMontesinos** [comment]
Hi,
as I mentioned in the other post, I tend not to relase the code as the dataloaders are not straight forward to use if not having an exact, identical copy of the training data, since they depends on IDs that are generated automatically.

I think you can pretty much use whichever framework (that uses complex mask) to get the results. 
The weights are available in the [Release section of the repository](https://github.com/JuanFMontesinos/VoViT/releases)

If it's really important I can try to release the code, but it's not as clean as this repo nor straight-forward to run.
